src/map.rs

```diff
@@ -906,7 +912,7 @@ where
 
                     next_table.store_bin(i, low_bin);
                     next_table.store_bin(i + n, high_bin);
-                    table.store_bin(i, table.get_moved(Shared::from(next_table_ptr), guard));
+                    table.store_bin(i, table.get_moved(next_table_ptr, guard));
```

Comment:
Hmm, this change doesn't seem right. Did the merge go wrong somewhere maybe?

Reply:
When I ran `cargo clippy` it came up with the following:
```
warning: useless conversion to the same type: `reclaim::Shared<raw::Table<K, V>>`
   --> src\map.rs:915:56
    |
915 |                     table.store_bin(i, table.get_moved(Shared::from(next_table_ptr), guard));
    |                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: consider removing `Shared::from()`: `next_table_ptr`
    |
    = note: `#[warn(clippy::useless_conversion)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#useless_conversion
```
I can revert the changes because I do realise that removing those conversions in this PR might be out of place.

Reply:
Oh, yeah, let me fix that real quick just directly on `master` :+1:

Reply:
Fixed on `master`.